.gitignore

```diff
@@ -12,5 +12,3 @@ openapi.yaml
 .env
 .DS_Store
 .run
```

Comment:
Does it make sense to leave `docker/volumes` in here so that people who ran the Docker setup before this change won't accidentally check in their database files?

Reply:
I thought I was the only one that used this 😅 ? Good call though